src/main/kotlin/com/vladsch/kotlin/jdbc/ModelProperties.kt

```diff
@@ -615,8 +615,18 @@ class ModelProperties<M>(val session: Session, val tableName: String, val dbCase
 
     fun listQuery(whereClause: String, params: Map<String, Any?>, alias: String? = null): SqlQuery {
         val sb = StringBuilder()
-        appendSelectSql(sb, alias)
-        appendSelectSql(sb).append(whereClause, alias)
+        val aliasedWhereClause = if(!alias.isNullOrEmpty()) {
+            whereClause.split(" ").joinToString(" ") {
```

Comment:
1. The where clause is a user provided string. I don't see why it should be changed instead of the user providing one already properly aliased. The alias parameter in this method is intended for the model generated select query.

2. You are splitting on spaces and comparing if the split part is a column name. So it will only work if the column is surrounded by spaces but that is not a requirement in SQL. So `column=value` will not be aliased.
    
     One solution here would be to go through all the properties and defaults and see if the part starts with a column name but then more checks would be needed to make sure that it is not a false positive match.

I do not see a use case for this because the where clause provided by the user should already have the columns aliased.